src/test/scala/LocalTest.scala

```diff
@@ -0,0 +1,856 @@
+import analysis.{AddressRange, DSC, DSG, DSN, DataLocation, HeapLocation}
+import ir.Endian.BigEndian
+import ir.{Assign, BVADD, BinaryExpr, BitVecLiteral, ConvertToSingleProcedureReturn, DirectCall, Memory, MemoryAssign, MemoryLoad, SharedMemory}
+import org.scalatest.funsuite.AnyFunSuite
+import test_util.TestUtil
+import ir.dsl.*
+import specification.Specification
+import util.{BASILConfig, BoogieGeneratorConfig, ILLoadingConfig, IRContext, RunUtils, StaticAnalysisConfig}
+
+class LocalTest extends AnyFunSuite, TestUtil {
```

Comment:
Rather than 'LocalTest' wouldn't the name DSATest be clearer for this file & class?